src/lib/string.ts

```diff
@@ -64,3 +64,47 @@ export function truncate(value: string | undefined, maxLength: number, suffix =
 
   return `${value.slice(0, maxLength)}${suffix}`;
 }
+
+/**
+ * Checks if the provided string is a valid swiss IBAN number
+ * @param ibanNumber The provided IBAN number to check
+ * Must be in one of the following formats:
+ * - "CHXX XXXX XXXX XXXX XXXX X" with whitespaces
+ * - "CHXXXXXXXXXXXXXXXXXXX" without whitespaces
+ * @returns The result of the IBAN number check
+ */
+export function isValidSwissIbanNumber(ibanNumber: string): boolean {
+  if (isNullOrWhitespace(ibanNumber)) {
+    return false;
+  }
+
+  const compactIbanNumberWithWhiteSpaces = new RegExp(/^CH\d{2}(?:\s?\d{4}){4}\s?\d{1}$/);

```

Comment:
Maybe it should be splitted in 2 regex, one for the version with the spaces and one without, otherwise it might consider valid also this "CH930076 20116238 5295 7"